README.md

```diff
@@ -304,4 +304,4 @@ DO NOT EDIT DIRECTLY.
 |:------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------|:--------------------------------|:-------------------------------------------------------------------------------------------------------------|:-------------------------------------------------|:-------------------------------------------|
 | [ds000117](https://github.com/bids-standard/bids-examples/tree/master/ds000117)                             | A multi-subject, multi-modal human neuroimaging dataset of 19 subjects on a MEG visual task | anat, beh, dwi, fmap, func, meg | T1w, beh, bold, channels, coordsystem, dwi, events, headshape, magnitude1, magnitude2, meg, phasediff, scans | [link](https://openneuro.org/datasets/ds000117/) | [@RikHenson](https://github.com/RikHenson) |
 | [eeg_ds003645s_hed_demo](https://github.com/bids-standard/bids-examples/tree/master/eeg_ds003645s_hed_demo) | Shows usage of Hierarchical Event Descriptor (HED) in .tsv files                            | eeg, beh                        | channels, eeg, events, participants, scans                                                                   | [link](https://openneuro.org/datasets/ds003645)  | [@VisLab](https://github.com/VisLab)       |
-| [synthetic](https://github.com/bids-standard/bids-examples/tree/master/synthetic)                           | A synthetic dataset                                                                         | anat, beh, func                 | T1w, beh, bold, events, physio, scans, sessions, stim                                                        | n/a                                              | [@effigies](https://github.com/effigies)   |
\ No newline at end of file
+| [synthetic](https://github.com/bids-standard/bids-examples/tree/master/synthetic)                           | A synthetic dataset                                                                         | anat, beh, func                 | T1w, beh, bold, events, physio, scans, sessions, stim                                                        | n/a                                              | [@effigies](https://github.com/effigies)   |
```

Comment:
I believe this table is generated automatically. If it will get reverted with the next regeneration, then it would be better to fix up the script.

Reply:
probably "no newline" at the end of file? https://github.com/bids-standard/bids-examples/blob/711be5ba05e8071eb9048aa07ce79c30dcecfdbd/tools/print_dataset_listing.py#L129

And it got auto-inserted by the editor the last person touching this file was using?